pages/common/git-filter-repo.md

```diff
@@ -6,7 +6,7 @@
 
 - Replace a sensitive string in all files:
 
-`git filter-repo --replace-text <(echo '{{find}}==>{{replacement}}')`
+`git filter-repo --replace-text {{path/to/file_with_string}}`
```

Comment:
This no longer documents the format of the sensitive string and its replacement.

Reply:
Alright, I'll revert the change and add a portability note.